src/uebung_3_3/BMIRechner.java

```diff
@@ -0,0 +1,165 @@
+package uebung_3_3;
+
+public class BMIRechner {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+		double koerperGroesse = 1.60, gewicht = 70.0;
+		double bmi;
+		int alter = 33;
+		boolean m�nnlich = false;
+
+		// Berechnung BMI
+		bmi = gewicht / Math.pow(koerperGroesse, 2);
+		System.out.println("Ihr BMI liegt bei " + bmi);
+
+		// Bewertung BMI
+		if (alter <= 24) {
+			if (m�nnlich == true) {
+				if (bmi < 20) {
+					System.out.println("Sie sind untergewichtig!");
+				}
+				if (bmi > 25) {
+					System.out.println("Sie sind �bergewichtig!");

```

Comment:
Hier in Zeichenketten sind Umlaute ok, da sie schlimmstenfalls auf der Konsole nicht ordentlich dargestellt werden würde. Perfekt wäre aber auch hier darauf zu verzichten.